src/EditorFeatures/CSharpTest/GenerateFromMembers/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersTests.cs

```diff
@@ -113,23 +120,26 @@ class Program
     string s;
     bool b;
 
-    public Program(int i)
+    public Program(int i, string s, bool b)
     {
         this.i = i;
+        this.s = s;
+        this.b = b;
     }
 
-    public Program(int i, string s, bool b) : this(i)
+    public Program(int i, string s) : this(i)
     {
         this.s = s;
-        this.b = b;
     }
 }");
         }
```

Comment:
this should have a sibling test added that shows what happens if you pick the test at index 1.